pint/__init__.py

```diff
@@ -75,6 +75,14 @@ def _build_unit(units):
     return _APP_REGISTRY.Unit(units)
 
 
+def get_application_registry():
+    """
+    Get the application registry which is used for unpickling operations.
+    """
+    assert isinstance(registry, UnitRegistry)
```

Comment:
maybe I am missing something, but... where is the `registry` variable defined?

Reply:
Obviously you're not and I was so focused on the pickle part that I did not even check the get_app_registry because it looked "so simple to do", and forgot to delete this line...